README.md

```diff
@@ -59,7 +58,7 @@ cd $MODULE
 gcc -Iinclude -std=gnu11 -o /tmp/tt-console scripts/tt-console/console.c
 
 # Build and flash firmware
-west build --sysbuild -p -b tt_blackhole@p100a/tt_blackhole/smc app/smc \
+west build --sysbuild -b tt_blackhole@p100a/tt_blackhole/smc app/smc \
```

Comment:
I'd keep the `-p`- if people have to retry this step I think that is helpful. Just my two cents

Reply:
Sure.
It shouldn't be necessary, but if it is, that probably means we aren't tracking build dependencies correctly. If that's the case, build dependencies should be fixed.

Reply:
It likely isn't necessary if they get the command right first try- what I've seen in the past is that people get into weird states (ie they type the board wrong on their first attempt or something)

Reply:
No, it isn't necessary on the first try, but it shouldn't be necessary on subsequent tries either.